mantle/kola/harness.go

```diff
@@ -1122,7 +1203,7 @@ func makeNonExclusiveTest(tests []*register.Test, flight platform.Flight) regist
 	}
 
 	nonExclusiveWrapper := register.Test{
-		Name: "non-exclusive-tests",
+		Name: fmt.Sprintf("non-exclusive-test-bucket%v", bucketNum),
```

Comment:
Easier to read on my eyes if we add an extra `-`:

```suggestion
		Name: fmt.Sprintf("non-exclusive-test-bucket-%v", bucketNum),
```